core/icons/comment_icon.ts

```diff
@@ -54,6 +54,9 @@ export class CommentIcon extends Icon implements IHasBubble, ISerializable {
   /** The text of this comment. */
   private text = '';
 
+  /** The text of this comment before the most recent change event. */
+  private oldText = '';
```

Comment:
I don't think you need to add a separate property for this. You should be able to use the current text as the old text before assigning the new text.

Something like:
```
const newText = this.textInputBubble.getText();
eventUtils.fire(this.text, newText);  // Actually use the right signature :P
this.text = newText;
```

Could you try that and verify that it behaves correctly?